Source/ProgressionSystemRuntime/Private/Data/PSWorldSubsystem.cpp

```diff
@@ -171,11 +160,47 @@ void UPSWorldSubsystem::OnWorldBeginPlay(UWorld& InWorld)
 void UPSWorldSubsystem::Deinitialize()
 {
 	Super::Deinitialize();
-	PSHUDComponentInternal = nullptr;
-	PSCurrentSpotComponentInternal = nullptr;
-	SaveGameDataInternal = nullptr;
-	ProgressionSettingsDataInternal.Empty();
-	StarDynamicProgressMaterial = nullptr;
+}
+
+// Invoked after a game feature plugin is unloaded
+void UPSWorldSubsystem::OnGameFeatureUnloading(const UGameFeatureData* GameFeatureData, const FString& PluginURL)
+{
+	DestroyStarActors();
+	IGameFeatureStateChangeObserver::OnGameFeatureUnloading(GameFeatureData, PluginURL);
+}
+
+// Invoked in the early stages of the game feature plugin loading phase
+void UPSWorldSubsystem::OnGameFeatureLoading(const UGameFeatureData* GameFeatureData, const FString& PluginURL)
+{
+	const FString ProgressionPlugin = TEXT("/ProgressionSystem/ProgressionSystem");
```

Comment:
Just `ProgressionSystem` would be enough, no need for slashes and second duplicated name

Reply:
done with: https://github.com/h4rdmol/ProgressionSystem/pull/26/commits/47260c66ff529013a615cf6f56a11af802480ec4